src/argilla/client/feedback/training/frameworks/trl.py

```diff
@@ -85,16 +86,17 @@ def __init__(
             TrainingTaskForDPO,
```

Comment:
I think we should update the docstring within the training task to allow for these examples?

Reply:
ohh sorry, maybe the `TrainingTask.for_*`-ones

Reply:
Same thing, they only use `dataset.prepare_for_training` in all examples